core/src/main/java/com/graphhopper/reader/dem/AbstractElevationProvider.java

```diff
@@ -1,14 +1,14 @@
 /*
  *  Licensed to GraphHopper GmbH under one or more contributor
- *  license agreements. See the NOTICE file distributed with this work for 
+ *  license agreements. See the NOTICE file distributed with this work for
  *  additional information regarding copyright ownership.
- * 
- *  GraphHopper GmbH licenses this file to you under the Apache License, 
- *  Version 2.0 (the "License"); you may not use this file except in 
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
  *  compliance with the License. You may obtain a copy of the License at
- * 
+ *
  *       http://www.apache.org/licenses/LICENSE-2.0
- * 
```

Comment:
I just updated IntelliJ, they added formatting for Copyright Statements (I mean who hasn't been waiting for this great feature). Even if I disable formatting for copyright statements and for java doc IntelliJ still messes this up...

Before merging I could just overwrite all copyright statements that got changed by the autoformatting?

Reply:
sure